packages/web-frontend/src/disaster/sagas.js

```diff
@@ -24,10 +24,7 @@ import { DISASTER } from '../containers/OverlayDiv/constants';
 import { formatDateForApi } from '../utils';
 import { selectCurrentOrgUnit } from '../selectors';
 
-// As a module that requires extra data for its dashboard item data fetches, the 'disaster' sagas
-// file must export this generator function to allow the global fetchDashboardItemData saga to
-// call it and wait for it to finish during the fetch of 'disaster' project dashboard elements
```

Comment:
I remove this comment. I believe it was required when the more abstract logic was in place, but not anymore since we explicitly call this function